src/main/java/racingcar/domain/Name.java

```diff
@@ -0,0 +1,27 @@
+package racingcar.domain;
+
+public record Name(String value) {
+
+    public Name {
+        value = nameTrim(value);
+        validateNotEmpty(value);
+        validationNameLength(value);
+    }
```

Comment:
검증을 하기 전에 value를 초기화 하신 이유가 있을까요?

Reply:
value에 trim을 주기 위함입니다!
isEmpty() 메소드가 " " 이와 같은 공백은 사이즈가 1로 인식을 하더라구요!

```suggestion
    public Name {
        validateNotEmpty(value.trim());
        validationNameLength(value.trim());
        value = nameTrim(value);
    }
```

흠 이렇게 하는게 좋았을라나요.. 고민이네요.. 뭔가 중복이 있는것 같기도하구요!
동균님은 어떻게 생각하세요?

Reply:
`view`에서 입력 받을 때 `trim`을 주는 것도 괜찮아 보이네요!

Reply:
https://github.com/woowacourse-precourse/java-racingcar-7/pull/456#discussion_r1822360865
> 제 개인적인 생각으로 View에서는 사용자의 입력만을 처리해야 한다고 생각합니다.

`view`에서 `trim`을 사용하지 않은 이유입니다!

Reply:
그러다면 `isBlank` 메서드를 사용해도 괜찮을 것 같아요!